source/Reloaded.Mod.Loader/Logging/LogWriter.cs

```diff
@@ -82,18 +86,33 @@ public void Flush()
     }
 
     [MethodImpl(MethodImplOptions.Synchronized)]
-    private void OnPrintMessage(object sender, string message)
+    private void OnWrite(object sender, (string text, Color color) e)
+    {
+        _writeCache.Append(e.text);
+    }
+
+    [MethodImpl(MethodImplOptions.Synchronized)]
+    private void OnWriteLine(object sender, (string text, Color color) e)
     {
         if (_logItems.Count == MaxBufferLength)
             Flush();
 
-        _logItems.Add($"[{DateTime.Now:HH:mm:ss}] {message}");
+        if(_writeCache.Length == 0)
```

Comment:
Just add a space here after `if` to meet style guidelines.
Aside from that, everything is a-ok.

Reply:
Oh my bad, missed that